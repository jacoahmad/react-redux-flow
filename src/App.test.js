import React from 'react';
import { shallow, mount, render } from 'enzyme';
import App from './App';

describe('App', () => {
  it('Should render without crashing', () => 
    expect(shallow(<App />)).toMatchSnapshot()
  );
});