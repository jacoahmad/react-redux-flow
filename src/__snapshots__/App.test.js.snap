// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Should render without crashing 1`] = `<Layout />`;
